repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
I guess that's the thing. If we aim for convention over configuration, then our documentation needs to be very clear.

Your explanation makes sense, but it feels like we're fighting the configuration system. In this case, I suppose it doesn't matter too much since, as you pointed out, virtually no user is disabling this feature. For a feature that is configured more often, though, would you make the same decision?

Anyway, I'm not going to lose sleep at night because this configuration doesn't give me warm fuzzies, so leaving it as written sounds like what we want to do.